docker/tangle-parachain/README.md

```diff
@@ -0,0 +1,47 @@
+# Tangle Parachain Docker
+
+## Installation Instructions
+
+A Tangle parachain node can be spun up quickly using Docker. At the time of writing, the Docker version used was 19.03.6. When connecting to testnet, it will take a few hours/days to completely sync the embedded relay chain. Make sure that your system meets the requirements.
+
+Create a local directory to store the chain data:
+
+```bash
+mkdir /var/lib/tangle/
+```
+
+
+## Run via CLI :
+
+You can use the following command to pull the latest image and run from your CLI, remember to set `YOUR-NODE-NAME` in two different places
+
+```bash
+docker run --platform linux/amd64 --network="host" -v "/var/lib/data" --entrypoint ./tangle-parachain \
+ghcr.io/webb-tools/tangle/tangle-parachain:main \
+--base-path=/data \
+--chain dev \
+--name="YOUR-NODE-NAME" \
+--execution wasm \
+--wasm-execution compiled \
+--state-pruning archive \
```

Comment:
I don't think we want to have `archive` node as the default instruction here. Setting up an `archive` node is a very intentional act?